src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -16,46 +20,57 @@ import FastImage from 'react-native-fast-image';
  *
  * @param {ProfilePicturePropsParam} props
  */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
+
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
+      </TouchableNativeFeedback>
     </View>
-}
+  );
+};
 
 let styles = StyleSheet.create({
-    addCircle: {position: 'absolute', top: 25, left: 25},
-    profileImage: {
-        width: 100,
-        height: 100,
-        borderRadius: 100,
-        // marginBottom: 12
-    },
-    profileImageContainer: {
-        width: 100,
-        borderRadius: 100,
-    },
-    wrapImageProfile: {
-        marginTop: 14,
-        flexDirection: 'column',
-    },
-})
+  addCircle: {position: 'absolute', top: 25, left: 25},
+  profileImage: {
+    width: 97,
+    height: 97,
+    borderRadius: 100,
+    marginLeft: 1.4,
+    marginTop: 2,
+    marginBottom: 2
+  },
+  profileImageContainer: {
+    width: 100,
+    borderRadius: 100
+    // zIndex: 1000
+  },
+  wrapImageProfile: {
+    // marginTop: 14,
+    flexDirection: 'column'
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes to the styles seem fine, adjusting the size and positioning of the profile image and its container. However, the commented out `zIndex: 1000` and `marginTop: 14` should be removed to keep the code clean.
```diff
-    // zIndex: 1000
+    
-    // marginTop: 14,
+    
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->